be/src/util/storage_backend_mgr.cpp

```diff
@@ -15,17 +15,17 @@
 // specific language governing permissions and limitations
 // under the License.
```

Comment:
Do not refactor storage_backend_mgr.cpp or .h， focus on refactor sequence file， not others.